PIL/_imaging.pyi

```diff
@@ -0,0 +1,91 @@
+from PIL.aliases import *
+from typing import Optional, List, Text, Any
+
+PILLOW_VERSION = ...
+DEFAULT_STRATEGY = ...
+FILTERED = ...
+HUFFMAN_ONLY = ...
+RLE = ...
+FIXED = ...
+
+# List of functions is incomplete
+
+def alpha_composite(core1: ImagingCore, core2: ImagingCore) -> ImagingCore: ...
```

Comment:
These `ImagingCore` types need to be enclosed in `''` or `""`, or the type definition (class) `ImagingeCore` needs to be moved to the top of the file.

Reply:
I don't understand why it runs without errors. In a normal program these types would have to be defined before you could use them, e.g. in the following little example.
```python
try:
    # breaks
    def bar(x: A) -> A:
        print(x)
        return x
except NameError as err:
    print(err)
    def bar(x: 'A') -> 'A':
        print(x)
        return x


class A:
    def __init__(self, val: int) -> int:
        self.val = val

    def __repr__(self) -> str:
        return f'A({self.val})'


# fine
def foo(x: A) -> A:
    print(x)
    return x


if __name__ == '__main__':
    bar(A(3))
    foo(A(2))

# A(3)
# A(2)
```
This isn't necessary in stub files? It could be a bug in PyCharm too; it highlights them as undefined.